### ISSUE TITLE
Can We Edit Crash Messages?

### ISSUE THREAD

**Issue by EQ2Alyza** [TurBoss/7h#28]
The crash messages give coding jargon to explain what caused the crash, so the average user doesn't understand what the message is telling them. Is it possible to edit those messages to give instructions on how to fix the issue?

**Comment by TurBoss** [comment]
sure I'll take a look

**Comment by TurBoss** [comment]
is there a well-know issues? with their messages?

**Comment by EQ2Alyza** [comment]
Yes, quite a few. I don't have them off the top of my head, so I'll have to compile them when I have some free time. I thought maybe they'd be stored in the source code for you to find, but I guess not.
